application/common/mail/welcome.html.php

```diff
@@ -0,0 +1,86 @@
+<?php
+use yii\helpers\Html as yHtml;
+
+/**
+ * @var string $employeeId
+ * @var string $firstName
+ * @var string $lastName
+ * @var string $displayName
+ * @var string $username
+ * @var string $email
+ * @var string $active
+ * @var string $locked
+ * @var string $lastChangedUtc
+ * @var string $lastSyncedUtc
+ * @var string $lastLoginUtc
+ * @var string $passwordExpiresUtc
+ * @var string $emailSignature
+ * @var string $helpCenterUrl
+ * @var string $idpDisplayName
+ * @var string $passwordForgotUrl
+ * @var string $passwordProfileUrl
+ * @var string $supportEmail
+ * @var string $supportName
+ * @var bool   $isMfaEnabled
+ */
+?>
+<p>
+    Dear <?=yHtml::encode($displayName)?>,
+</p>
+<p>
+    Congratulations! You have logged into your new <?=yHtml::encode($idpDisplayName)?> account for the first time.
+    <?=yHtml::encode($idpDisplayName)?> is in the process of transitioning from logging into websites using
+    an Insite account to this new "<?=yHtml::encode($idpDisplayName)?> account". Starting in January 2018 the option
+    to login to websites using your Insite account will go away. This new <?=yHtml::encode($idpDisplayName)?>
```

Comment:
Minor tweak: `to login to websites` --> `to log in to websites` (since we've been making that distinction in other communications).